benchmarks/src/benches/mod.rs

```diff
@@ -26,4 +26,3 @@ pub mod iter_pedersen;
 pub mod iter_sha2;
 pub mod membership;
 pub mod sudoku;
-pub mod zeth;
```

Comment:
If we remove this, then:

https://github.com/risc0/risc0/blob/29c3cd6bdd68d96fa908cab0401efc71602daa9b/benchmarks/src/main.rs#L127C5-L130C6

would fail to compile